### PR TITLE
Ragdoll authoring part twelve

### PR DESCRIPTION
**Pull request by greerdv**
## What does this PR do?
Separates ragdoll authoring viewport UI buttons into separate clusters for collider editing and joint limit editing. Adds tooltips for the buttons. 

## How was this PR tested?
Played around with animation editor.

### CODE REVIEW

**Review by jonawals — APPROVED** [review]
LGTM

**Review by hultonha — APPROVED** [review]
LGTM! 👍

**Comment by hultonha** [comment]
@greerdv any chance of a screen shot to demo this change too? 🙂

File: Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupViewportUiCluster.h [hunk 1]
```diff
@@ -39,15 +39,35 @@ namespace EMotionFX
...
+        struct ButtonData
+        {
+            AzToolsFramework::ViewportUi::ClusterId m_clusterId;
+            AzToolsFramework::ViewportUi::ButtonId m_buttonId;
+        };
+
+        constexpr static const char* const ColliderTranslationTooltip = "Switch to collider translation mode";
+        constexpr static const char* const ColliderRotationTooltip = "Switch to collider rotation mode";
+        constexpr static const char* const ColliderDimensionsTooltip = "Switch to collider dimensions mode";
+        constexpr static const char* const JointLimitParentRotationTooltip = "Switch to joint limit parent frame rotation mode";
```
Comment (amzn-jillich) [inline Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsSetupViewportUiCluster.h:52]: Nit: What about moving these just to where the tooltips are passed as function parameters? We probably don't have a second use case for them and the member types are quite verbose.

**Review by amzn-jillich — APPROVED** [review]

**Review by moraaar — APPROVED** [review]
LGTM

**Comment by greerdv** [comment]
https://user-images.githubusercontent.com/82226198/182615890-615da3bb-d531-4dd6-b21a-e4fecd0553da.mp4

**Pull request merged**